compiler-plugin/src/main/kotlin/arrow/meta/quotes/element/CatchClause.kt

```diff
@@ -0,0 +1,38 @@
+package arrow.meta.quotes.element
+
+import arrow.meta.quotes.Scope
+import arrow.meta.quotes.nameddeclaration.stub.Parameter
+import org.jetbrains.kotlin.psi.KtCatchClause
+import org.jetbrains.kotlin.psi.KtExpression
+
+/**
+ * <code> """ catch $`(parameter)` $`{ body }` """.catchClause </code>
+ *
+ * A template destructuring [Scope] for a [KtCatchClause].
+ *
+ * ```kotlin:ank:silent
+ * import arrow.meta.Meta
+ * import arrow.meta.Plugin
+ * import arrow.meta.invoke
+ * import arrow.meta.quotes.Transform
+ * import arrow.meta.quotes.catchClause
+ *
+ * val Meta.reformatCatchClause: Plugin
+ *  get() =
+ *   "ReformatCatchClause" {
+ *    meta(
+ *     catchClause({ true }) { c ->
+ *      Transform.replace(
+ *       replacing = c,
+ *       newDeclaration = """ catch $`(parameter)` $`{ body }` """.catch
+ *      )
+ *     }
+ *    )
+ *   }
+ *```
+ */
+class CatchClause(
+  override val value: KtCatchClause?,
+  val `(parameter)`: Parameter = Parameter(value?.catchParameter),
+  val `{ body }`: Scope<KtExpression> = Scope(value?.catchBody)
```

Comment:
@bloderxd dumb question but does this actually format in the strings like the above?

Reply:
Yeah, all kind of kt elements has its text property and that's what scope does in its `toString` function: invoke that property, I'm just wondering if we should keep calling it as `{ body }`, or change it to `body`